docs/getting-credentials-and-accounts.md

```diff
@@ -0,0 +1,55 @@
+2. An AMS Role defined through the OCM Resources mechanism has been defined, available for developers of the KAS Fleet Manager to get the needed permissions
+   to perform KAS Fleet Manager development. The definition of the role can be found in 
+   https://gitlab.cee.redhat.com/service/uhc-account-manager/-/blob/master/pkg/api/roles/managed_kafka_service.go.
+3. Users interested in developing KAS Fleet Manager should have this role assigned to their OCM account. This can be done using the OCM Resources
+   mechanism. An example of assigning the AMS role previously mentioned to a user can be found [here](https://gitlab.cee.redhat.com/service/ocm-resources/-/merge_requests/812).
+4. KAS Fleet Manager developers also need AMS quota for installing specific OCM add-ons that KAS Fleet Manager requires. An example showing the required quotas for the addons can be found [here](https://gitlab.cee.redhat.com/service/ocm-resources/-/blob/master/data/uhc-stage/orgs/13640203.yaml).
+
+# Vault
+For automation (fleet-manager deployments and app-interface pipelines) it is required to use [vault](https://gitlab.cee.redhat.com/service/app-interface#manage-secrets-via-app-interface-openshiftnamespace-1yml-using-vault) for secrets management
+
+# AWS
+AWS accounts are required for provisioning development OSD clusters and to access AWS Route53 for Domain Name System. To request a AWS accounts, have a chat with your team's manager.
+> NOTE, if you are in the Middleware, make sure to send an email to `mw-billing-leaders@redhat.com` request the account to be created. The email should follow the below format:
+```
+*Request Type*
+New AWS account
+*Team*
+<Whoever the requesting team is>
+
+*Cost*
+<Calculations for the estimated monthly charges>
+*Why?*
+<The reason for requesting the account>
+```
+> NOTE
+Within the AWS account used to provision OSD clusters, you must create an osdCcsAdmin IAM user with the following requirements:
+- This user needs at least Programmatic access enabled.
+- This user must have the AdministratorAccess policy attached to it.
+See https://docs.openshift.com/dedicated/osd_planning/aws-ccs.html#ccs-aws-customer-procedure_aws-ccs for more information.
+
+# SSO authentication
+Depending on whether interacting with public or private endpoints, the authentication can be set up as follows:
+* Red Hat SSO sso.redhat.com for public endpoints
+* Red HAT SSO or MAS SSO for private endpoints. See this
+[doc](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/mas-sso/service-architecture/service-architecture.md)
+to understand how kas-fleet-manager is using MAS-SSO for authentication
+
+See [feature-flags](feature-flags.md#Keycloak) to understand flags used for authentication.
+
+## sso.redhat.com service account
+To avail of all required OCM services the fleet-manager depends on, it is
```

Comment:
Let's note that this isn't required as people can use their own offline tokens for development. 

This is only necessary if it's required to have a dedicated service account for running KFM.

Reply:
Updated